src/main/features/response/views/confirmation/full-defence-steps.njk

```diff
@@ -2,13 +2,17 @@
 
 {% set claimantName = claim.claimData.claimant.name %}
 
-<h2 class="heading-small">{{ t('If you agreed to try mediation') }}</h2>
-
-<p>{{ t('We’ll ask {{ claimantName }} if they want to try mediation.', { claimantName: claimantName }) | safe }}</p>
-<p>{{ t('If they agree we’ll contact you to arrange an appointment. If not, we’ll tell you what to do.') }}</p>
-
-<h2 class="heading-small">{{ t('If you chose not to try mediation') }}</h2>
-<p>{{ t('We’ll contact you to tell you what to do.') }}</p>
+{% if claim.response.freeMediation === FreeMediationOption.YES %}
+  <p>{{ t('We’ll ask {{ claimantName }} if they want to try mediation.', { claimantName: claimantName }) | safe }}</p>
+  <p>{{ t('If they agree, we’ll contact you with a date for an appointment. If not, we’ll tell you what to do.') }}</p>
```

Comment:
The content doesn't seem to match the prototype in https://tools.hmcts.net/jira/browse/ROC-5614.
is this correct ?